index.html

```diff
@@ -1,11 +1,133 @@
+              <h6 class="author-name__post">Shaun Willis on
+              <a class="date-time" href="#">Monday, October 7th 9:22pm</a></h6>
+            </div>
+          </div>
+          <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+             sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+             laborum.</p>
+          <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+             sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+              .</p>
+             <h4 class="post-quote">| I wish I knew that than!</h4>
+          <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+             sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim
+              ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip
+              laborum.</p>
+          <p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+               ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip.</p>
+              <div class="container__btn">
+                <button class="btn"><span class="read-comment fa fa-1x fa-comment-o">Read Comments</span></button>
+                <button class="btn"><span class="read-comment fa fa-1x fa-cloud-upload">Read Later</span></button>
+                <button class="btn__selected"><span class="read-comment fa fa-1x fa-share-square-o">Share This Post</span></button>
+              </div>
+        </div>
+      </div>
+      <div class="container__sidebar">
+        <div class="sidebar">
+          <img src="img/Shaun-Willis.jpg" alt="" class="author-photo">
+          <h6 class="author-name">Shaun Willis</h6>
+          <ul class="social__media-list">
+            <li class="social__media-item">Twitter</li>
+            <li class="social__media-item">Facebook</li>
+            <li class="social__media-item">Instagram</li>
+          </ul>
+          <div class="image__sidebar">
+            <img src="img/paddling.jpeg" alt="" class="image__sidebar-item">
+            <a href="#" class="ad-link">About this ad.</a>
+            <div class="container__sidebar-content">
+              <div class="sidebar-content">
+                <h6 class="sidebar-content__title">Paddling Lessons</h6>
+                <p class="sidebar-content__plaintext">Lorem ipsum dolor sit
+                   amet, consectetur adipisicing elit, sed do eiusmod tempor
+                   incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+                    veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+
+<hr class="divider">
```

Comment:
This can be easier accomplished by putting a border top on the footer section.

Either way it should span across the full width of the page.